docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -91,6 +87,88 @@ spec:
     - ssh-rsa AAAAB3NzaC1yc2... jwjones@833efcab1482.home.example.com
 ```
 
+<pre>
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+   name: my-cluster-name             <a href="#name-required"># Name of the cluster (required)</a>
+spec:
+   clusterNetwork:                   <a href="#clusternetwork-required"># Cluster network configuration (required)</a>
+      cniConfig:                     <a href="#clusternetworkcniconfig-required"># Cluster CNI plugin - default: cilium (required)</a>
+         cilium: {}
+      pods:
+         cidrBlocks:                 <a href="#clusternetworkpodscidrblocks0-required"># Subnet CIDR notation for pods (required)</a>
+            - 192.168.0.0/16
+      services:
+         cidrBlocks:                 <a href="#clusternetworkservicescidrblocks0-required"># Subnet CIDR notation for services (required)</a>
+            - 10.96.0.0/12
+   controlPlaneConfiguration:        <a href="#controlplaneconfiguration-required"># Specific cluster control plane config (required)</a>
+      count: <span style="color:green">2</span>                       <a href="#controlplaneconfigurationcount-required"># Number of control plane nodes (required)</a>
+      endpoint:                      <a href="#controlplaneconfigurationendpointhost-required"># IP for control plane endpoint (required)</a>
+         host: <span>"192.168.0.10"</span>
+      machineGroupRef:               <a href="#controlplaneconfigurationmachinegroupref-required"># baremetal-specific Kubernetes node config (required)</a>
+        kind: TinkerbellMachineConfig
+        name: my-cluster-name
+   datacenterRef:                    <a href="#datacenterref"># Kubernetes object with baremetal-specific config </a>
+      kind: TinkerbellDatacenterConfig  
+      name: my-cluster-datacenter
+   kubernetesVersion: <span>"1.28"</span>         <a href="#kubernetesversion-required"># Kubernetes version to use for the cluster (required)</a>
+   workerNodeGroupConfigurations:    <a href="#workernodegroupconfigurations"># List of node groups you can define for workers (required) </a>
+   - count: <span style="color:green">2</span>                        <a href="#workernodegroupconfigurationscount"># Number of worker nodes </a>
+     machineGroupRef:                <a href="#workernodegroupconfigurationsmachinegroupref-required"># baremetal-specific Kubernetes node objects (required) </a>
+       kind: TinkerbellMachineConfig
+       name: my-cluster-name
+     name: md-0                      <a href="#workernodegroupconfigurationsname-required"># Name of the worker nodegroup (required) </a>
+---
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: TinkerbellDatacenterConfig
+metadata:
+   name: my-cluster-datacenter
```

Comment:
Check if you want this to say "my-cluster-datacenter". In previous versions, this was the same name as the cluster name, which was "my-cluster-name".